go.mod

```diff
@@ -3,21 +3,21 @@ module github.com/rhobs/multicluster-observability-addon
 go 1.20
 
 require (
-	github.com/imdario/mergo v0.3.12
+	github.com/cert-manager/cert-manager v1.13.3
```

Comment:
Can we depend only on their APIs if they possibly provide a API specific module?

Reply:
Unfortunately they don't seem to offer one :( https://github.com/cert-manager/cert-manager/tree/master/pkg/apis